views/components/BarGraph/index.jsx

```diff
@@ -30,52 +43,99 @@ const BarGraph = ({ graph }) => {
   }
 
   return (
-    <ResponsiveContainer width="100%" height={500}>
-      <BarChart width={600} height={400} data={sanitizeSiteData(siteData)}>
-        <Legend
-          verticalAlign="top"
-          height={50}
-          margin={{ top: 0, left: 0, right: 0, bottom: 20 }}
-        />
-        <XAxis
-          dataKey="name"
-          height={100}
-          angle={55}
-          textAnchor="start"
-          interval={0}
-        />
-        <YAxis width={80}>
-          <Label value="Percent" angle={-90} />
-        </YAxis>
-        <Tooltip
-          content={<BarGraphTooltip studyTotals={graph.studyTotals} />}
-        />
-        {graph.labels.map((label) => {
-          return (
-            <Bar
-              key={label.name}
-              dataKey={`percentages[${label.name}]`}
-              name={label.name}
-              id={label.name}
-              stackId="a"
-              maxBarSize={80}
-              fill={label.color}
-            >
-              <LabelList
-                valueAccessor={(entry) => {
-                  const percent = entry.percentages[label.name]
+    <Paper
+      sx={{
+        width: '100%',
+      }}
+    >
+      <ResponsiveContainer width="100%" height={500}>
+        <BarChart
+          id="recharts-wrapper"
+          data={sanitizeSiteData(siteData)}
+          barSize={50}
+          margin={{ top: 50 }}
+          onMouseMove={(data) => {
+            if (data.isTooltipActive) {
+              const chartWidth = Number(
+                document.getElementById('chartWidth').attributes.width.value
```

Comment:
Why are we using `document.getElementById`? If we need to tap the DOM to get this value I think we should use a `ref` so that we don't query the DOM on every React render.

Reply:
good catch
[f1fdf68](https://github.com/AMP-SCZ/dpdash/pull/683/commits/f1fdf681e593a6459f2d272d1dcee587aa88217c)